src/test/java/sg/dex/starfish/impl/operations/FindPrime_JsonI_JsonO.java

```diff
@@ -11,24 +11,32 @@
 import java.util.Map;
 
 /**
- * This class is a sample implementation of Invoke Service.
- * This class can be refereed how to implement an Operation interface .
- * It just do calculation of prime number based on argument passed
+ * This class is a memory implementation of Invoke Service.
+ * It calculate all prime number present before any given number.
+ * It reads metadata from a file which has basic detail of the input and output type.
  */
 public class FindPrime_JsonI_JsonO extends AMemoryOperation implements Operation {
     protected FindPrime_JsonI_JsonO(String meta, MemoryAgent memoryAgent) {
         super(meta, memoryAgent);
     }
 
     public static FindPrime_JsonI_JsonO create(MemoryAgent memoryAgent) throws IOException {
-        // read metadata
-        String asset_metaData = new String(Files.readAllBytes(Paths.get("src/test/resources/assets/test_metadata.json")));
+        // read metadata from a file
+        //this metadata for an operation that find all prime number  before a given input
+        String asset_metaData =
```

Comment:
Use a FileAsset or a ResourceAsset here rather than reading the file content directly. We have our tests to make sure that Assets work correctly!